.github/workflows/issue.yml

```diff
@@ -0,0 +1,13 @@
+name: Labeling new issue
+on:
+  issues:
+      types: ['opened']
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: Renato66/auto-label@master
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          labels-synonyms: '{"GSSOC21":["gssoc","gssoc21","GSSOC21","GSSOC","GSSoC21"]}'
```

Comment:
Try making it the following:
```yaml
labels-synonyms: '{"GSSOC21":["gssoc","gssoc21","GSSOC21","GSSOC","GSSoC''21"]}'
```
Test it and confirm if it works.

Reply:
> Try making it the following:
> 
> ```yaml
> labels-synonyms: '{"GSSOC21":["gssoc","gssoc21","GSSOC21","GSSOC","GSSoC''21"]}'
> ```
> 
> Test it and confirm if it works.

It works absolutely fine. Thank you for the information